README.md

```diff
@@ -1,38 +1,74 @@
 # IWD dev team ladder
 
+This job ladder exposes all roles inside IWD development department, and define the goals, expectations and responsibilities of each role.
+Constant progression, and meeting your individual objectives will help you to climb it.
+
+Be aware that some basic expectations are not specified in the roles:
+* Being involved into team reviews
+* Delivering software on-time
+* Communicating, seeking help when stuck
+* Being a "team player"
+
 ## Developer roles
```

Comment:
If you want to be fancy, you can say "Individual contributor track or IC track" see https://medium.com/@adrian.chung/the-individual-contributor-track-vs-the-management-track-7b22cf85b749

Reply:
The way I understand this article is that the IC track is the ladder a IC can follow.

Being a team player (i.e. helping others, communicating, understanding/following established rules, ...) is a different point to me.

Reply:
I'm talking about Developer roles --> Individual Contributor